pkg/controller/kieapp/webconsole.go

```diff
@@ -0,0 +1,141 @@
+package kieapp
+
+//go:generate go run -mod=vendor ./defaults/.packr/packr.go
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/ghodss/yaml"
+	"github.com/gobuffalo/packr/v2"
+	"strconv"
+	"strings"
+
+	consolev1 "github.com/openshift/api/console/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	TargetNamespace   = "openshift-console"
+	ConsoleYAMLSample = "ConsoleYAMLSample"
+)
+
+type CustomResourceDefinition struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+}
+
+func CreateConsoleYAMLSamples(r *Reconciler) {
```

Comment:
This should be package private, so start with lowercase: `create`

Reply:
modified